cypress/integration/Toast.spec.ts

```diff
@@ -0,0 +1,93 @@
+import * as h from '../helpers';
+
+function statusToast() {
+  return cy.findByRole('status');
+}
+
+function dialogToast() {
+  return cy.findByRole('dialog');
+}
+
+function getToastCloseButton() {
+  return cy.get('[aria-label="Close"');
+}
+
+function getToastActionButton() {
+  return cy.get('[aria-label="View more details"');
```

Comment:
I'm wondering if there's a better way to get this element.

Reply:
https://kentcdodds.com/blog/making-your-ui-tests-resilient-to-change